api/v1/topologymap.go

```diff
@@ -76,3 +79,19 @@ func (m *NodeTopologyMap) GetKeyValues(topologyKey string) (string, []string) {
 
 	return topologyKey, values
 }
+
+// GetKeyValues returns a node label matching the topologyKey and all values for StrechCluster
+// for that label across all storage nodes
```

Comment:
Please update the comment as per the working of this function.

Reply:
done